sfcasts/ep5-security/entry-point.md

```diff
@@ -33,7 +33,7 @@ we'll talk about soon - *do* provide an entry point... and we'll see that.
 
 ## Making our Authenticator an Entry Point
 
-But anyways, none of our authenticators provide an entry point... so let's add one!
+But anyway, none of our authenticators provide an entry point... so let's add one!
```

Comment:
"But anyways" is exactly what Ryan says in the video, and we do want our scripts to match the video. Could you revert this change?

Reply:
Change reverted. 😉
Please accept my apologies for the mistake.

Reply:
No problem! Thanks for reverting it :)